CreeDictionary/API/models.py

```diff
@@ -356,10 +361,57 @@ def fetch_lemma_by_user_query(user_query: str, **kwargs) -> "CreeAndEnglish":
         user_query = user_query.lower()
 
         # build up result_lemmas in 2 ways
-        # 1. spell relax in descriptive fst
+        # 1. affix search (return all results that ends/starts with the query string)
+        # 2. spell relax in descriptive fst
         # 2. definition containment of the query word
+
         cree_results: Set[CreeResult] = set()
 
+        if (
+            len(user_query) > 4
```

Comment:
Use the configurable constant `CREE_DICTIONARY_AFFIX_SEARCH_LENGTH` or whatever you decide to name it.

Reply:
So... I guess there are two constants:

 - min word length for affix search?
 - affix length?

Oh dear.